src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -28,6 +28,12 @@ BEACON_DISCOVERY_PORT_NUM	= 9000
 BEACON_HTTP_PORT_NUM		= 4000
 BEACON_METRICS_PORT_NUM		= 5054
 
+# The min/max CPU/memory that the beacon node can use
```

Comment:
rather than having these declared per launcher can we declare these in `input_parser`?

Reply:
As different clients would need to use different defaults, I thought its better to do it per client.

Reply:
can this be resolved?